test/unit/tasks/ReportSpec.scala

```diff
@@ -49,9 +49,9 @@ class ReportSpec extends AnyWordSpec with Matchers with TryValues {
       result.get should equal(
         Seq(
           "url\tlanguage\tserviceName\tserviceAbsoluteUrl\tcontactFrontendServiceId\tcomplianceStatus\tproblemCount\tmilestoneCount\tearliestMilestoneDate\tautomatedTestingOnly\tstatementVisibility\tserviceLastTestedDate\tstatementCreatedDate\tstatementLastUpdatedDate\tstatementType\tBusiness Area\tDDC\tLive or Classic\ttype of Service",
-          "https://www.qa.tax.service.gov.uk/accessibility-statement/test-service\ten\tTest (English)\thttps://www.tax.service.gov.uk/test/some.test.service\tsome.contact-frontend\tfull\t0\t0\t\tfalse\tpublic\t2020-02-28\t2020-03-15\t2020-05-01\tHMRC\t\t\t\t",
-          "https://www.qa.tax.service.gov.uk/accessibility-statement/test-service\tcy\tTest (Welsh)\thttps://www.tax.service.gov.uk/test/some.test.service\tsome.contact-frontend\tfull\t0\t0\t\tfalse\tpublic\t2020-02-28\t2020-03-15\t2020-05-01\tHMRC\t\t\t\t",
-          "https://www.qa.tax.service.gov.uk/accessibility-statement/english-service\ten\tEnglish Only\thttps://www.tax.service.gov.uk/test/some.test.service\tsome.contact-frontend\tfull\t0\t0\t\tfalse\tpublic\t2020-02-28\t2020-03-15\t2020-05-01\tHMRC\t\t\t\t",
+          "https://www.qa.tax.service.gov.uk/accessibility-statement/test-service\ten\tTest (English)\thttps://www.tax.service.gov.uk/test/some.test.service\tsome.contact-frontend\tfull\t0\t0\t1900-01-01\tfalse\tpublic\t2020-02-28\t2020-03-15\t2020-05-01\tHMRC\t\t\t\t",
+          "https://www.qa.tax.service.gov.uk/accessibility-statement/test-service\tcy\tTest (Welsh)\thttps://www.tax.service.gov.uk/test/some.test.service\tsome.contact-frontend\tfull\t0\t0\t1900-01-01\tfalse\tpublic\t2020-02-28\t2020-03-15\t2020-05-01\tHMRC\t\t\t\t",
+          "https://www.qa.tax.service.gov.uk/accessibility-statement/english-service\ten\tEnglish Only\thttps://www.tax.service.gov.uk/test/some.test.service\tsome.contact-frontend\tfull\t0\t0\t1900-01-01\tfalse\tpublic\t2020-02-28\t2020-03-15\t2020-05-01\tHMRC\t\t\t\t",
```

Comment:
looks like all these cover `earliestMilestoneDate`  - is there a test covering `lastTestedDate`?